rskj-core/src/test/java/org/ethereum/datasource/DataSourceWithCacheTest.java

```diff
@@ -40,6 +40,16 @@ public void getAfterMiss() {
         dataSourceWithCache.get(randomKey);
         dataSourceWithCache.get(randomKey);
 
+        try (DataSourceKeyIterator iterator = dataSourceWithCache.keyIterator()){
+
+            iterator.seekToFirst();
+
+            assertTrue(iterator.hasNext());
+            assertArrayEquals(iterator.next(), randomKey);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
I guess we could do `Assert.fail(...)` here in case we get an exception?

Reply:
Addressed